examples/actions/add-headers.mdx

```diff
@@ -1,13 +1,75 @@
 import ConfigExample from "../../src/components/ConfigExample.tsx";
+import CelTemplateSyntaxExplanation from "/src/components/CelTemplateSyntaxExplanation";
 
 export const type = "add-headers";
-export const config = {
-	headers: {
-		"is-ngrok": "1",
-		country: "${conn.geo.country_code}",
-	},
-};
 
 export const ExampleHTTP = () => (
-	<ConfigExample config={{ actions: [{ type, config }] }} />
+	<ConfigExample
+		config={{
+			actions: [
+				{
+					type,
+					config: {
+						headers: {
+							"is-ngrok": "1",
+							country: "${conn.geo.country_code}",
+						},
+					},
+				},
+			],
+		}}
+	/>
 );
+
+export const TemplatingExamples = () => (
+	<>
+		<ConfigExample
+			snippetText={"simple variable replacement"}
+			config={{
+				actions: [
+					{
+						type,
+						config: {
+							headers: {
+								time: "${time.now}",
+							},
+						},
+					},
+				],
+			}}
+		/>
+		<ConfigExample
+			snippetText={"using a macro"}
+			config={{
+				actions: [
+					{
+						type,
+						config: {
+							headers: {
+								"x-secure-session": "getReqCookie('session').Secure",
```

Comment:
does this example work? I think `getReqCookie()` returns an array

Reply:
It _did_, but now that you mention it I know we change this recently. I might use a different macro for safety / double check it 👀